src-test/Main.hs

```diff
@@ -48,6 +48,12 @@ main = do
           hPutStrLn stderr "unexpected arguments passed to yaml2event sub-command"
           exitFailure
 
+    ("check":args')
```

Comment:
let's call this `yaml2yaml-validate` instead; as it's yet another yaml-to-yaml mode

```suggestion
    ("yaml2yaml-validate":args')
```